.github/workflows/ubuntu.yml

```diff
@@ -55,78 +55,107 @@ jobs:
       #
       - name: Build environment setup
         run: |
-          mkdir _build
+          mkdir build
```

Comment:
Is this necessary ? We like having those build and temporary folders starting with _ to distinguish them easily.

Reply:
no, it isn't, it is just what the autogen script suggests and what for example clangd expects as a build directory without configuration - I revert that part